examples/nfs/storageclass.yaml

```diff
@@ -51,5 +51,10 @@ parameters:
   # This option is available with Ceph v17.2.6 and newer.
   # secTypes: <sectype-list>
 
+  # (optional) List of IP addresses, hostnames or IPv4 network addresses that
+  # that these export permissions apply to. The <client-list> is a comma
```

Comment:
Let's word it similar to the commit message and make it clear that only these clients will be able to access
the export ?